ni_measurementlink_generator/tests/test_assets/example_renders/SampleMeasurement.serviceconfig

```diff
@@ -5,7 +5,12 @@
       "serviceClass": "SampleMeasurement_Python",
       "descriptionUrl": "https://www.example.com/SampleMeasurement.html",
       "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
-      "path": "start.bat"
+      "path": "start.bat",
+      "annotations": {
+        "ni/service.description": "",
+        "ni/service.collection": "",
+        "ni/service.tags": []
```

Comment:
I'm confused as to why the tags are typed as Optional[str] just like the other two, but we treat them as an array here.

Reply:
It is because on the @ClickOption, I set it as multiple but each is entered as a string.

Reply:
I see that Multiple=True now.

Reply:
I see how this works. See my comment on the serviceconfig template.